File: 100-Days-Series/Day 61/PalindromicPatternWithNumber.java
class PalindromicPatternWithNumber{
    // Fucntion to print Palindromic Pattern with numbers
    public static void getPrintPalindromicNumber(int rows){
        for(int i=1;i<=rows;i++){
            // Print Spaces
            for(int j=1;j<=rows-i;j++){
                System.out.print("  ");
            }
            // Print Increasing numbers
            for(int j=1;j<=i;j++){
                System.out.print(j+" ");
            }
            // Print Decreasing numbers
            for(int j=i-1;j>=1;j--){
                System.out.print(j+" ");
            }
            System.out.println();
        }
    }
    // Main Method 
    public static void main(String [] args){
        int rows;
        java.util.Scanner sc=new java.util.Scanner(System.in);
        System.out.print("Enter the number of rows : ");
        rows=sc.nextInt();
        getPrintPalindromicNumber(rows);    
    }
}

File: 100-Days-Series/Day 61/PalindromicPatternWithNumber.cpp
#include<iostream>
using namespace std;
// Function to print Palindromic Pattern With number
void getPrintPalindromic(int rows){
    for(int i=1;i<=rows;i++){
        // Print Spaces
        for(int j=1;j<=rows-i;j++){
            cout<<"  ";
        }
        // Print Increading numbers
        for(int j=1;j<=i;j++){
            cout<<j<<" ";
        }
        // Print Decreasing numbers
        for(int j=i-1;j>=1;--j){
            cout<<j<<" ";
        }
        cout<<endl;
    }
}
// Let's Start Here
int main(){
    int rows;
    cout<<"\nEnter the number of rows : ";
    cin>>rows;
    getPrintPalindromic(rows);
}